cypress/support/metaTestHelper.js

```diff
@@ -6,6 +6,18 @@ export const testResponseCode = (path, responseCode) => {
   });
 };
 
+export const retrieveCookieValue = (cookieName, value) => {
```

Comment:
This could be called `expectCookieValue` or something to make its functionality clearer.